// ember-cli-mirage/jekyll-tmp/dist/assets/dummy.js
"use strict";

/* jshint ignore:start */



/* jshint ignore:end */

define('dummy/adapters/application', ['exports', 'active-model-adapter'], function (exports, _activeModelAdapter) {
  exports['default'] = _activeModelAdapter['default'];
});
define('dummy/adapters/word-smith', ['exports', 'ember-data'], function (exports, _emberData) {
  var JSONAPIAdapter = _emberData['default'].JSONAPIAdapter;
  exports['default'] = JSONAPIAdapter;
});
define('dummy/app', ['exports', 'ember', 'dummy/resolver', 'ember-load-initializers', 'dummy/config/environment'], function (exports, _ember, _dummyResolver, _emberLoadInitializers, _dummyConfigEnvironment) {
  var Application = _ember['default'].Application;

  _ember['default'].MODEL_FACTORY_INJECTIONS = true;

  var App = Application.extend({
    modulePrefix: _dummyConfigEnvironment['default'].modulePrefix,
    podModulePrefix: _dummyConfigEnvironment['default'].podModulePrefix,
    Resolver: _dummyResolver['default']
  });

  (0, _emberLoadInitializers['default'])(App, _dummyConfigEnvironment['default'].modulePrefix);

  exports['default'] = App;
});
define('dummy/components/app-version', ['exports', 'ember-cli-app-version/components/app-version', 'dummy/config/environment'], function (exports, _emberCliAppVersionComponentsAppVersion, _dummyConfigEnvironment) {

  var name = _dummyConfigEnvironment['default'].APP.name;
  var version = _dummyConfigEnvironment['default'].APP.version;

  exports['default'] = _emberCliAppVersionComponentsAppVersion['default'].extend({
    version: version,
    name: name
  });
});
define('dummy/controllers/application', ['exports', 'ember'], function (exports, _ember) {
  var Controller = _ember['default'].Controller;
  exports['default'] = Controller.extend();
});
define('dummy/controllers/array', ['exports', 'ember'], function (exports, _ember) {
  exports['default'] = _ember['default'].Controller;
});
define('dummy/controllers/object', ['exports', 'ember'], function (exports, _ember) {
  exports['default'] = _ember['default'].Controller;
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/assert.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/assert.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/assert.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/assert.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/assert.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/assert.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/db-collection.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/db-collection.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/db-collection.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/db-collection.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/db-collection.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/db-collection.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/db.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/db.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/db.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/db.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/db.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/db.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/factory.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/factory.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/factory.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/factory.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/factory.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/factory.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/faker.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/faker.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/faker.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/faker.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/faker.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/faker.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/index.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/index.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/index.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/index.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/index.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/index.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/associations/association.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/orm/associations/association.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/associations/association.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/associations/association.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/orm/associations/association.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/associations/association.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/associations/belongs-to.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/orm/associations/belongs-to.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/associations/belongs-to.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/associations/belongs-to.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/orm/associations/belongs-to.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/associations/belongs-to.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/associations/has-many.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/orm/associations/has-many.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/associations/has-many.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/associations/has-many.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/orm/associations/has-many.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/associations/has-many.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/collection.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/orm/collection.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(false, 'modules/ember-cli-mirage/orm/collection.js should pass jscs.\nrequireArrayDestructuring: Use array destructuring at modules/ember-cli-mirage/orm/collection.js :\n    92 |   */\n    93 |  remove(model) {\n    94 |    let match = this.models.filter(m => _isEqual(m.attrs, model.attrs))[0];\n--------------------------------------------------------------------------------^\n    95 |    if (match) {\n    96 |      let i = this.models.indexOf(match);');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/collection.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/orm/collection.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/collection.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/model.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/orm/model.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(false, 'modules/ember-cli-mirage/orm/model.js should pass jscs.\nrequireCommentsToIncludeAccess: You must supply `@public`, `@private`, or `@protected` for block comments. at modules/ember-cli-mirage/orm/model.js :\n   328 |  }\n   329 |\n   330 |  /**\n----------^\n   331 |   * Originally we validated this via association.setId method, but it triggered\n   332 |   * recursion. That method is designed for updating an existing model\'s ID so\nrequireObjectDestructuring: Property assignments should use destructuring at modules/ember-cli-mirage/orm/model.js :\n   406 |\n   407 |  _disassociateFromOldInverses(association) {\n   408 |    let key = association.key;\n-------------------------^\n   409 |    let fk = association.getForeignKey();\n   410 |    let inverse = association.inverse();\nrequireObjectDestructuring: Property assignments should use destructuring at modules/ember-cli-mirage/orm/model.js :\n   425 |\n   426 |  _saveNewChildren(association) {\n   427 |    let key = association.key;\n-------------------------^\n   428 |    let fk = association.getForeignKey();\n   429 |    let tempChildren = this._tempAssociations && this._tempAssociations[key];');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/model.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/orm/model.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(false, 'modules/ember-cli-mirage/orm/model.js should pass jshint.\nmodules/ember-cli-mirage/orm/model.js: line 398, col 11, \'inverse\' is defined but never used.\nmodules/ember-cli-mirage/orm/model.js: line 399, col 11, \'foo\' is defined but never used.\nmodules/ember-cli-mirage/orm/model.js: line 7, col 8, \'BelongsTo\' is defined but never used.\nmodules/ember-cli-mirage/orm/model.js: line 8, col 8, \'_isEqual\' is defined but never used.\n\n4 errors');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/schema.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/orm/schema.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(false, 'modules/ember-cli-mirage/orm/schema.js should pass jscs.\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   117 |  new(type, attrs) {\n   118 |\n   119 |      // let fk = foreignKeysHash[attr];\n--------^\n   120 |      // debugger;\n   121 |      // assert(\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   118 |\n   119 |      // let fk = foreignKeysHash[attr];\n   120 |      // debugger;\n--------^\n   121 |      // assert(\n   122 |      //   !fk || this.schema.db[toCollectionName(association.modelName)].find(fk),\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   119 |      // let fk = foreignKeysHash[attr];\n   120 |      // debugger;\n   121 |      // assert(\n--------^\n   122 |      //   !fk || this.schema.db[toCollectionName(association.modelName)].find(fk),\n   123 |      //   `Couldn\\\'t find ${association.modelName} with id = ${fk}`\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   120 |      // debugger;\n   121 |      // assert(\n   122 |      //   !fk || this.schema.db[toCollectionName(association.modelName)].find(fk),\n--------^\n   123 |      //   `Couldn\\\'t find ${association.modelName} with id = ${fk}`\n   124 |      // );\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   121 |      // assert(\n   122 |      //   !fk || this.schema.db[toCollectionName(association.modelName)].find(fk),\n   123 |      //   `Couldn\\\'t find ${association.modelName} with id = ${fk}`\n--------^\n   124 |      // );\n   125 |\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   122 |      //   !fk || this.schema.db[toCollectionName(association.modelName)].find(fk),\n   123 |      //   `Couldn\\\'t find ${association.modelName} with id = ${fk}`\n   124 |      // );\n--------^\n   125 |\n   126 |      // this[attr] = fk;\nvalidateIndentation: Expected indentation of 4 characters at modules/ember-cli-mirage/orm/schema.js :\n   124 |      // );\n   125 |\n   126 |      // this[attr] = fk;\n--------^\n   127 |    return this._instantiateModel(dasherize(type), attrs);\n   128 |  }');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/orm/schema.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/orm/schema.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/orm/schema.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/response.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/response.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/response.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/response.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/response.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/response.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handler.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handler.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handler.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handler.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handler.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handler.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/base.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/base.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/base.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/base.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/base.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/base.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/function.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/function.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/function.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/function.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/function.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/function.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/object.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/object.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/object.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/object.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/object.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/object.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/base.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/shorthands/base.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/base.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/base.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/shorthands/base.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/base.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/delete.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/shorthands/delete.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/delete.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/delete.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/shorthands/delete.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/delete.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/get.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/shorthands/get.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/get.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/get.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/shorthands/get.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/get.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/head.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/shorthands/head.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/head.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/head.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/shorthands/head.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/head.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/post.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/shorthands/post.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/post.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/post.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/shorthands/post.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/post.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/put.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/route-handlers/shorthands/put.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/put.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/route-handlers/shorthands/put.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/route-handlers/shorthands/put.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/route-handlers/shorthands/put.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializer-registry.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/serializer-registry.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializer-registry.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializer-registry.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/serializer-registry.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializer-registry.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializer.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/serializer.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializer.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializer.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/serializer.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializer.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializers/active-model-serializer.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/serializers/active-model-serializer.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializers/active-model-serializer.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializers/active-model-serializer.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/serializers/active-model-serializer.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializers/active-model-serializer.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializers/json-api-serializer.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/serializers/json-api-serializer.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializers/json-api-serializer.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializers/json-api-serializer.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/serializers/json-api-serializer.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializers/json-api-serializer.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializers/rest-serializer.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/serializers/rest-serializer.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializers/rest-serializer.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/serializers/rest-serializer.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/serializers/rest-serializer.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/serializers/rest-serializer.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/server.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/server.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/server.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/server.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/server.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/server.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/trait.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/trait.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/trait.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/trait.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/trait.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/trait.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/extend.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/utils/extend.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/extend.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/extend.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/utils/extend.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/extend.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/inflector.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/utils/inflector.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/inflector.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/inflector.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/utils/inflector.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/inflector.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/normalize-name.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/utils/normalize-name.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/normalize-name.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/normalize-name.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/utils/normalize-name.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/normalize-name.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/read-modules.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/utils/read-modules.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/read-modules.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/read-modules.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/utils/read-modules.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/read-modules.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/reference-sort.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/utils/reference-sort.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/reference-sort.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/reference-sort.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/utils/reference-sort.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/reference-sort.js should pass jshint.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/uuid.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - modules/ember-cli-mirage/utils/uuid.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/uuid.js should pass jscs.');
  });
});
define('dummy/ember-cli-mirage/tests/modules/ember-cli-mirage/utils/uuid.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - modules/ember-cli-mirage/utils/uuid.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'modules/ember-cli-mirage/utils/uuid.js should pass jshint.');
  });
});
define('dummy/helpers/pluralize', ['exports', 'ember-inflector/lib/helpers/pluralize'], function (exports, _emberInflectorLibHelpersPluralize) {
  exports['default'] = _emberInflectorLibHelpersPluralize['default'];
});
define('dummy/helpers/singularize', ['exports', 'ember-inflector/lib/helpers/singularize'], function (exports, _emberInflectorLibHelpersSingularize) {
  exports['default'] = _emberInflectorLibHelpersSingularize['default'];
});
define("dummy/initializers/active-model-adapter", ["exports", "active-model-adapter", "active-model-adapter/active-model-serializer"], function (exports, _activeModelAdapter, _activeModelAdapterActiveModelSerializer) {
  exports["default"] = {
    name: 'active-model-adapter',
    initialize: function initialize() {
      var application = arguments[1] || arguments[0];
      application.register('adapter:-active-model', _activeModelAdapter["default"]);
      application.register('serializer:-active-model', _activeModelAdapterActiveModelSerializer["default"]);
    }
  };
});
define('dummy/initializers/app-version', ['exports', 'ember-cli-app-version/initializer-factory', 'dummy/config/environment'], function (exports, _emberCliAppVersionInitializerFactory, _dummyConfigEnvironment) {
  exports['default'] = {
    name: 'App Version',
    initialize: (0, _emberCliAppVersionInitializerFactory['default'])(_dummyConfigEnvironment['default'].APP.name, _dummyConfigEnvironment['default'].APP.version)
  };
});
define('dummy/initializers/container-debug-adapter', ['exports', 'ember-resolver/container-debug-adapter'], function (exports, _emberResolverContainerDebugAdapter) {
  exports['default'] = {
    name: 'container-debug-adapter',

    initialize: function initialize() {
      var app = arguments[1] || arguments[0];

      app.register('container-debug-adapter:main', _emberResolverContainerDebugAdapter['default']);
      app.inject('container-debug-adapter:main', 'namespace', 'application:main');
    }
  };
});
define('dummy/initializers/data-adapter', ['exports', 'ember'], function (exports, _ember) {

  /*
    This initializer is here to keep backwards compatibility with code depending
    on the `data-adapter` initializer (before Ember Data was an addon).
  
    Should be removed for Ember Data 3.x
  */

  exports['default'] = {
    name: 'data-adapter',
    before: 'store',
    initialize: _ember['default'].K
  };
});
define('dummy/initializers/ember-cli-mirage', ['exports', 'ember-cli-mirage/utils/read-modules', 'dummy/config/environment', 'dummy/mirage/config', 'ember-cli-mirage/server', 'lodash/object/assign'], function (exports, _emberCliMirageUtilsReadModules, _dummyConfigEnvironment, _dummyMirageConfig, _emberCliMirageServer, _lodashObjectAssign) {
  exports.startMirage = startMirage;
  exports['default'] = {
    name: 'ember-cli-mirage',
    initialize: function initialize(application) {
      if (arguments.length > 1) {
        // Ember < 2.1
        var container = arguments[0],
            application = arguments[1];
      }

      if (_shouldUseMirage(_dummyConfigEnvironment['default'].environment, _dummyConfigEnvironment['default']['ember-cli-mirage'])) {
        startMirage(_dummyConfigEnvironment['default']);
      }
    }
  };

  function startMirage() {
    var env = arguments.length <= 0 || arguments[0] === undefined ? _dummyConfigEnvironment['default'] : arguments[0];

    var environment = env.environment;
    var modules = (0, _emberCliMirageUtilsReadModules['default'])(env.modulePrefix);
    var options = (0, _lodashObjectAssign['default'])(modules, { environment: environment, baseConfig: _dummyMirageConfig['default'], testConfig: _dummyMirageConfig.testConfig });

    return new _emberCliMirageServer['default'](options);
  }

  function _shouldUseMirage(env, addonConfig) {
    var userDeclaredEnabled = typeof addonConfig.enabled !== 'undefined';
    var defaultEnabled = _defaultEnabled(env, addonConfig);

    return userDeclaredEnabled ? addonConfig.enabled : defaultEnabled;
  }

  /*
    Returns a boolean specifying the default behavior for whether
    to initialize Mirage.
  */
  function _defaultEnabled(env, addonConfig) {
    var usingInDev = env === 'development' && !addonConfig.usingProxy;
    var usingInTest = env === 'test';

    return usingInDev || usingInTest;
  }
});
define('dummy/initializers/ember-data', ['exports', 'ember-data/setup-container', 'ember-data/-private/core'], function (exports, _emberDataSetupContainer, _emberDataPrivateCore) {

  /*
  
    This code initializes Ember-Data onto an Ember application.
  
    If an Ember.js developer defines a subclass of DS.Store on their application,
    as `App.StoreService` (or via a module system that resolves to `service:store`)
    this code will automatically instantiate it and make it available on the
    router.
  
    Additionally, after an application's controllers have been injected, they will
    each have the store made available to them.
  
    For example, imagine an Ember.js application with the following classes:
  
    App.StoreService = DS.Store.extend({
      adapter: 'custom'
    });
  
    App.PostsController = Ember.ArrayController.extend({
      // ...
    });
  
    When the application is initialized, `App.ApplicationStore` will automatically be
    instantiated, and the instance of `App.PostsController` will have its `store`
    property set to that instance.
  
    Note that this code will only be run if the `ember-application` package is
    loaded. If Ember Data is being used in an environment other than a
    typical application (e.g., node.js where only `ember-runtime` is available),
    this code will be ignored.
  */

  exports['default'] = {
    name: 'ember-data',
    initialize: _emberDataSetupContainer['default']
  };
});
define('dummy/initializers/es2015-polyfills', ['exports'], function (exports) {
  exports.initialize = initialize;

  function initialize() /* container, application */{
    // Production steps of ECMA-262, Edition 6, 22.1.2.1
    // Reference: https://people.mozilla.org/~jorendorff/es6-draft.html#sec-array.from
    if (!Array.from) {
      Array.from = (function () {
        var toStr = Object.prototype.toString;
        var isCallable = function isCallable(fn) {
          return typeof fn === 'function' || toStr.call(fn) === '[object Function]';
        };
        var toInteger = function toInteger(value) {
          var number = Number(value);
          if (isNaN(number)) {
            return 0;
          }
          if (number === 0 || !isFinite(number)) {
            return number;
          }
          return (number > 0 ? 1 : -1) * Math.floor(Math.abs(number));
        };
        var maxSafeInteger = Math.pow(2, 53) - 1;
        var toLength = function toLength(value) {
          var len = toInteger(value);
          return Math.min(Math.max(len, 0), maxSafeInteger);
        };

        // The length property of the from method is 1.
        return function from(arrayLike /*, mapFn, thisArg */) {
          // 1. Let C be the this value.
          var C = this;

          // 2. Let items be ToObject(arrayLike).
          var items = Object(arrayLike);

          // 3. ReturnIfAbrupt(items).
          if (arrayLike == null) {
            throw new TypeError('Array.from requires an array-like object - not null or undefined');
          }

          // 4. If mapfn is undefined, then let mapping be false.
          var mapFn = arguments.length > 1 ? arguments[1] : void undefined;
          var T = undefined;
          if (typeof mapFn !== 'undefined') {
            // 5. else
            // 5. a If IsCallable(mapfn) is false, throw a TypeError exception.
            if (!isCallable(mapFn)) {
              throw new TypeError('Array.from: when provided, the second argument must be a function');
            }

            // 5. b. If thisArg was supplied, let T be thisArg; else let T be undefined.
            if (arguments.length > 2) {
              T = arguments[2];
            }
          }

          // 10. Let lenValue be Get(items, "length").
          // 11. Let len be ToLength(lenValue).
          var len = toLength(items.length);

          // 13. If IsConstructor(C) is true, then
          // 13. a. Let A be the result of calling the [[Construct]] internal method of C with an argument list containing the single item len.
          // 14. a. Else, Let A be ArrayCreate(len).
          var A = isCallable(C) ? Object(new C(len)) : new Array(len);

          // 16. Let k be 0.
          var k = 0;
          // 17. Repeat, while k < len… (also steps a - h)
          var kValue = undefined;
          while (k < len) {
            kValue = items[k];
            if (mapFn) {
              A[k] = typeof T === 'undefined' ? mapFn(kValue, k) : mapFn.call(T, kValue, k);
            } else {
              A[k] = kValue;
            }
            k += 1;
          }
          // 18. Let putStatus be Put(A, "length", len, true).
          A.length = len;
          // 20. Return A.
          return A;
        };
      })();
    }
  }

  exports['default'] = {
    name: 'es2015-polyfills',
    initialize: initialize
  };
});
define('dummy/initializers/export-application-global', ['exports', 'ember', 'dummy/config/environment'], function (exports, _ember, _dummyConfigEnvironment) {
  exports.initialize = initialize;

  function initialize() {
    var application = arguments[1] || arguments[0];
    if (_dummyConfigEnvironment['default'].exportApplicationGlobal !== false) {
      var value = _dummyConfigEnvironment['default'].exportApplicationGlobal;
      var globalName;

      if (typeof value === 'string') {
        globalName = value;
      } else {
        globalName = _ember['default'].String.classify(_dummyConfigEnvironment['default'].modulePrefix);
      }

      if (!window[globalName]) {
        window[globalName] = application;

        application.reopen({
          willDestroy: function willDestroy() {
            this._super.apply(this, arguments);
            delete window[globalName];
          }
        });
      }
    }
  }

  exports['default'] = {
    name: 'export-application-global',

    initialize: initialize
  };
});
define('dummy/initializers/injectStore', ['exports', 'ember'], function (exports, _ember) {

  /*
    This initializer is here to keep backwards compatibility with code depending
    on the `injectStore` initializer (before Ember Data was an addon).
  
    Should be removed for Ember Data 3.x
  */

  exports['default'] = {
    name: 'injectStore',
    before: 'store',
    initialize: _ember['default'].K
  };
});
define('dummy/initializers/store', ['exports', 'ember'], function (exports, _ember) {

  /*
    This initializer is here to keep backwards compatibility with code depending
    on the `store` initializer (before Ember Data was an addon).
  
    Should be removed for Ember Data 3.x
  */

  exports['default'] = {
    name: 'store',
    after: 'ember-data',
    initialize: _ember['default'].K
  };
});
define('dummy/initializers/transforms', ['exports', 'ember'], function (exports, _ember) {

  /*
    This initializer is here to keep backwards compatibility with code depending
    on the `transforms` initializer (before Ember Data was an addon).
  
    Should be removed for Ember Data 3.x
  */

  exports['default'] = {
    name: 'transforms',
    before: 'store',
    initialize: _ember['default'].K
  };
});
define("dummy/instance-initializers/ember-data", ["exports", "ember-data/-private/instance-initializers/initialize-store-service"], function (exports, _emberDataPrivateInstanceInitializersInitializeStoreService) {
  exports["default"] = {
    name: "ember-data",
    initialize: _emberDataPrivateInstanceInitializersInitializeStoreService["default"]
  };
});
define('dummy/mirage/config', ['exports', 'ember-cli-mirage', 'ember'], function (exports, _emberCliMirage, _ember) {
  exports.testConfig = testConfig;
  var isBlank = _ember['default'].isBlank;

  exports['default'] = function () {
    // Contacts
    this.get('/contacts');
    // this.get('/contacts', ['contacts', 'addresses']);
    this.get('/contacts/:id');
    this.post('/contacts');
    this.put('/contacts/:id');
    this.del('/contacts/:id');

    // Friends
    this.get('/friends', { coalesce: true });

    // Pets
    this.get('/pets', function (_ref) {
      var db = _ref.db;

      return { pets: db.pets.filter(function (pet) {
          return pet.alive;
        }) };
    });

    this.post('/pets', function (_ref2, req) {
      var db = _ref2.db;

      var _JSON$parse = JSON.parse(req.requestBody);

      var pet = _JSON$parse.pet;

      if (isBlank(pet.name)) {
        var body = { errors: { name: ["can't be blank"] } };
        return new _emberCliMirage['default'].Response(422, { some: 'header' }, body);
      } else {
        return { pet: db.pets.insert(pet) };
      }
    });

    this.put('/pets/:id', function (_ref3, req) {
      var db = _ref3.db;

      var _JSON$parse2 = JSON.parse(req.requestBody);

      var pet = _JSON$parse2.pet;

      db.pets.update(pet.id, pet);
      return pet;
    });

    this['delete']('/pets/:id', function (_ref4, req) {
      var db = _ref4.db;
    }, 200);

    this.get('/word-smiths/:id');
  };

  function testConfig() {
    this.get('/friends/:id');
  }
});
define('dummy/mirage/factories/address', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage['default'].Factory.extend({});
});
define('dummy/mirage/factories/blog-post', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.Factory.extend({

    title: function title() {
      return _emberCliMirage.faker.lorem.sentence();
    }

  });
});
define('dummy/mirage/factories/contact', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage['default'].Factory.extend({
    title: _emberCliMirage.faker.list.cycle('Duke', 'Developer', 'Artist'),
    name: _emberCliMirage.faker.name.firstName,
    age: 20,

    email: function email(i) {
      return 'person' + i + '@test.com';
    },

    admin: function admin() {
      return this.age > 30;
    }

  });
});
define('dummy/mirage/factories/friend', ['exports', 'dummy/mirage/factories/contact'], function (exports, _dummyMirageFactoriesContact) {
  exports['default'] = _dummyMirageFactoriesContact['default'].extend({
    isYoung: function isYoung() {
      return this.age < 18;
    }
  });
});
define('dummy/mirage/factories/pet', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {

  var names = ['Rex', 'Toby', 'Sam', 'Andy', 'Lassie', 'Annibal', 'Spark'];

  exports['default'] = _emberCliMirage['default'].Factory.extend({
    alive: true,

    name: function name(i) {
      return names[i % names.length];
    }
  });
});
define('dummy/mirage/factories/word-smith', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.Factory.extend({

    name: function name() {
      return _emberCliMirage.faker.name.firstName() + ' ' + _emberCliMirage.faker.name.lastName();
    }

  });
});
define('dummy/mirage/fixtures/addresses', ['exports'], function (exports) {
  exports['default'] = [{
    id: 1,
    name: '123 Hyrule Way',
    contact_id: 1
  }, {
    id: 2,
    name: 'Mount Doom',
    contact_id: 2
  }];
});
define('dummy/mirage/fixtures/blog-posts', ['exports'], function (exports) {
  exports['default'] = [{
    id: 1,
    title: 'Lorem',
    wordSmithId: 1
  }, {
    id: 2,
    title: 'Ipsum',
    wordSmithId: 1
  }, {
    id: 3,
    title: 'Dolor',
    wordSmithId: 1
  }];
});
define('dummy/mirage/fixtures/contacts', ['exports'], function (exports) {
  exports['default'] = [{
    id: 1,
    name: 'Joe',
    address_ids: [1]
  }, {
    id: 2,
    name: 'Bob'
  }, {
    id: 3,
    name: 'Susan'
  }];
});
define('dummy/mirage/fixtures/friends', ['exports'], function (exports) {
  exports['default'] = [{
    id: 1,
    name: 'Joe',
    age: 10,
    isYoung: true
  }, {
    id: 2,
    name: 'Bob',
    age: 80,
    isYoung: false
  }];
});
define('dummy/mirage/fixtures/word-smiths', ['exports'], function (exports) {
  exports['default'] = [{
    id: 1,
    name: 'Link'
  }];
});
define('dummy/mirage/models/blog-post', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.Model.extend({});
});
define('dummy/mirage/models/contact', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.Model.extend({});
});
define('dummy/mirage/models/friend', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.Model.extend({});
});
define('dummy/mirage/models/word-smith', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.Model.extend({

    blogPosts: (0, _emberCliMirage.hasMany)()

  });
});
define('dummy/mirage/scenarios/default', ['exports'], function (exports) {
  exports['default'] = function (server) {
    var joe = server.create('contact', { name: 'Joe' });
    server.create('address', { name: '123 Hyrule Way', contact_id: joe.id });

    var bob = server.create('contact', { name: 'Bob' });
    server.create('address', { name: 'Mount Doom', contact_id: bob.id });
    server.create('contact', { name: 'Susan' });

    server.create('friend', { name: 'Joe', age: 10, isYoung: true });
    server.create('friend', { name: 'Bob', age: 80, isYoung: false });

    server.create('pet', { name: 'Pinky', alive: true });
    server.create('pet', { name: 'Bobby', alive: true });
    server.create('pet', { name: 'Brownie', alive: true });

    server.create('pet', { name: 'Cactus', alive: false });
  };
});
define('dummy/mirage/serializers/application', ['exports', 'ember-cli-mirage/serializers/active-model-serializer'], function (exports, _emberCliMirageSerializersActiveModelSerializer) {
  exports['default'] = _emberCliMirageSerializersActiveModelSerializer['default'];
});
define('dummy/mirage/serializers/blog-post', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.JSONAPISerializer.extend({});
});
define('dummy/mirage/serializers/word-smith', ['exports', 'ember-cli-mirage'], function (exports, _emberCliMirage) {
  exports['default'] = _emberCliMirage.JSONAPISerializer.extend({
    include: ['blogPosts']
  });
});
define('dummy/models/address', ['exports', 'ember-data'], function (exports, _emberData) {
  var Model = _emberData['default'].Model;
  var attr = _emberData['default'].attr;
  var belongsTo = _emberData['default'].belongsTo;
  exports['default'] = Model.extend({
    street: attr('string'),

    contact: belongsTo('contact')
  });
});
define('dummy/models/blog-post', ['exports', 'ember-data'], function (exports, _emberData) {
  var Model = _emberData['default'].Model;
  var attr = _emberData['default'].attr;
  var belongsTo = _emberData['default'].belongsTo;
  exports['default'] = Model.extend({

    title: attr(),

    wordSmith: belongsTo()

  });
});
define('dummy/models/contact', ['exports', 'ember-data'], function (exports, _emberData) {
  var Model = _emberData['default'].Model;
  var attr = _emberData['default'].attr;
  var belongsTo = _emberData['default'].belongsTo;
  exports['default'] = Model.extend({
    name: attr('string'),
    age: attr('number'),
    email: attr('string'),

    address: belongsTo('address')
  });
});
define('dummy/models/friend', ['exports', 'ember-data', 'dummy/models/contact'], function (exports, _emberData, _dummyModelsContact) {
  var attr = _emberData['default'].attr;
  exports['default'] = _dummyModelsContact['default'].extend({
    isYoung: attr('boolean')
  });
});
define('dummy/models/pet', ['exports', 'ember-data'], function (exports, _emberData) {
  var Model = _emberData['default'].Model;
  var attr = _emberData['default'].attr;
  exports['default'] = Model.extend({
    name: attr('string'),
    alive: attr('boolean')
  });
});
define('dummy/models/word-smith', ['exports', 'ember-data'], function (exports, _emberData) {
  var Model = _emberData['default'].Model;
  var attr = _emberData['default'].attr;
  var hasMany = _emberData['default'].hasMany;
  exports['default'] = Model.extend({

    name: attr(),

    blogPosts: hasMany()

  });
});
define('dummy/resolver', ['exports', 'ember-resolver'], function (exports, _emberResolver) {
  exports['default'] = _emberResolver['default'];
});
define('dummy/router', ['exports', 'ember', 'dummy/config/environment'], function (exports, _ember, _dummyConfigEnvironment) {
  var Router = _ember['default'].Router;
  exports['default'] = Router.extend({
    location: _dummyConfigEnvironment['default'].locationType
  });

  Router.map(function () {
    this.route('contacts', { path: '/' });
    this.route('contact', { path: '/:contact_id' });
    this.route('edit', { path: '/:contact_id/edit' });

    this.route('friends');
    this.route('friend', { path: '/friends/:friend_id' });
    this.route('close-friends');
    this.route('pets');

    this.route('word-smith', { path: '/word-smiths/:word_smith_id' });
  });
});
define('dummy/routes/application', ['exports', 'ember'], function (exports, _ember) {
  var Route = _ember['default'].Route;
  exports['default'] = Route.extend({

    actions: {
      createContact: function createContact() {
        var controller = this.controllerFor('contacts');
        var name = controller.get('newName');
        var newContact = this.store.createRecord('contact', { name: name });
        controller.set('newName', '');

        return newContact.save();
      },

      saveContact: function saveContact(contact) {
        var _this = this;

        return contact.save().then(function (contact) {
          _this.transitionTo('contact', contact);
        });
      },

      deleteContact: function deleteContact(contact) {
        var _this2 = this;

        return contact.destroyRecord().then(function () {
          _this2.transitionTo('contacts');
        });
      }
    }

  });
});
define('dummy/routes/close-friends', ['exports', 'ember'], function (exports, _ember) {
  var Route = _ember['default'].Route;
  exports['default'] = Route.extend({

    model: function model() {
      var store = this.get('store');
      return store.query('friend', { ids: [1, 3] }).then(function () {
        // I request 2 friends and then return all friends to be sure no other friend
        // was loaded into the store.
        return store.peekAll('friend');
      });
    }

  });
});
define('dummy/routes/contacts', ['exports', 'ember'], function (exports, _ember) {
  var Route = _ember['default'].Route;
  exports['default'] = Route.extend({

    model: function model() {
      var _this = this;

      return this.store.findAll('contact')['catch'](function (reason) {
        var errorMsg = reason.responseJSON ? reason.responseJSON.errors[0] : reason.errors[0];

        _this.set('error', errorMsg);
      });
    },

    setupController: function setupController(controller, model) {
      if (this.get('error')) {
        controller.set('error', this.get('error'));
      } else {
        controller.set('model', model);
      }
    }

  });
});
define('dummy/routes/friends', ['exports', 'ember'], function (exports, _ember) {
  var Route = _ember['default'].Route;
  exports['default'] = Route.extend({

    model: function model() {
      return this.store.findAll('friend');
    }

  });
});
define('dummy/routes/pets', ['exports', 'ember'], function (exports, _ember) {
  var Route = _ember['default'].Route;
  exports['default'] = Route.extend({

    actions: {
      createPet: function createPet() {
        var controller = this.controllerFor('pets');
        var name = controller.get('petName');
        controller.set('petName', '');
        this.store.createRecord('pet', { name: name, alive: true });
      }
    },

    setupController: function setupController(controller, model) {
      if (this.get('error')) {
        controller.set('error', this.get('error'));
      } else {
        controller.set('model', model);
      }
    },

    model: function model() {
      var _this = this;

      return this.store.findAll('pet')['catch'](function (reason) {
        var errorMsg = reason.responseJSON ? reason.responseJSON.errors[0] : reason.errors[0];
        _this.set('error', errorMsg);
      });
    }
  });
});
define('dummy/serializers/application', ['exports', 'active-model-adapter'], function (exports, _activeModelAdapter) {
  exports['default'] = _activeModelAdapter.ActiveModelSerializer;
});
define('dummy/serializers/blog-post', ['exports', 'ember-data'], function (exports, _emberData) {
  var JSONAPISerializer = _emberData['default'].JSONAPISerializer;
  exports['default'] = JSONAPISerializer;
});
define('dummy/serializers/word-smith', ['exports', 'ember-data'], function (exports, _emberData) {
  var JSONAPISerializer = _emberData['default'].JSONAPISerializer;
  exports['default'] = JSONAPISerializer;
});
define('dummy/services/ajax', ['exports', 'ember-ajax/services/ajax'], function (exports, _emberAjaxServicesAjax) {
  Object.defineProperty(exports, 'default', {
    enumerable: true,
    get: function get() {
      return _emberAjaxServicesAjax['default'];
    }
  });
});
define("dummy/templates/application", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["multiple-nodes", "wrong-type"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 4,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/application.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createElement("h2");
        dom.setAttribute(el1, "id", "title");
        var el2 = dom.createTextNode("Welcome to Ember");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var morphs = new Array(1);
        morphs[0] = dom.createMorphAt(fragment, 2, 2, contextualElement);
        return morphs;
      },
      statements: [["content", "outlet", ["loc", [null, [3, 0], [3, 10]]]]],
      locals: [],
      templates: []
    };
  })());
});
define("dummy/templates/close-friends", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    var child0 = (function () {
      return {
        meta: {
          "fragmentReason": false,
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 1,
              "column": 0
            },
            "end": {
              "line": 3,
              "column": 0
            }
          },
          "moduleName": "dummy/templates/close-friends.hbs"
        },
        isEmpty: false,
        arity: 1,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createTextNode("  ");
          dom.appendChild(el0, el1);
          var el1 = dom.createElement("p");
          var el2 = dom.createComment("");
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode(" is your ");
          dom.appendChild(el1, el2);
          var el2 = dom.createComment("");
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode("-year-old friend.");
          dom.appendChild(el1, el2);
          dom.appendChild(el0, el1);
          var el1 = dom.createTextNode("\n");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
          var element0 = dom.childAt(fragment, [1]);
          var morphs = new Array(2);
          morphs[0] = dom.createMorphAt(element0, 0, 0);
          morphs[1] = dom.createMorphAt(element0, 2, 2);
          return morphs;
        },
        statements: [["content", "friend.name", ["loc", [null, [2, 5], [2, 20]]]], ["content", "friend.age", ["loc", [null, [2, 29], [2, 43]]]]],
        locals: ["friend"],
        templates: []
      };
    })();
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["wrong-type"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 4,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/close-friends.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var morphs = new Array(1);
        morphs[0] = dom.createMorphAt(fragment, 0, 0, contextualElement);
        dom.insertBoundary(fragment, 0);
        dom.insertBoundary(fragment, null);
        return morphs;
      },
      statements: [["block", "each", [["get", "model", ["loc", [null, [1, 8], [1, 13]]]]], [], 0, null, ["loc", [null, [1, 0], [3, 9]]]]],
      locals: [],
      templates: [child0]
    };
  })());
});
define("dummy/templates/contact", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    var child0 = (function () {
      return {
        meta: {
          "fragmentReason": false,
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 6,
              "column": 2
            },
            "end": {
              "line": 8,
              "column": 2
            }
          },
          "moduleName": "dummy/templates/contact.hbs"
        },
        isEmpty: false,
        arity: 0,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createTextNode("    ");
          dom.appendChild(el0, el1);
          var el1 = dom.createElement("button");
          var el2 = dom.createTextNode("Edit this contact");
          dom.appendChild(el1, el2);
          dom.appendChild(el0, el1);
          var el1 = dom.createTextNode("\n");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes() {
          return [];
        },
        statements: [],
        locals: [],
        templates: []
      };
    })();
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["wrong-type", "multiple-nodes"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 14,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/contact.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createElement("p");
        var el2 = dom.createTextNode("The contact is ");
        dom.appendChild(el1, el2);
        var el2 = dom.createComment("");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createElement("div");
        var el2 = dom.createTextNode("\n");
        dom.appendChild(el1, el2);
        var el2 = dom.createComment("");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createElement("div");
        var el2 = dom.createTextNode("\n  ");
        dom.appendChild(el1, el2);
        var el2 = dom.createElement("button");
        var el3 = dom.createTextNode("Delete this contact");
        dom.appendChild(el2, el3);
        dom.appendChild(el1, el2);
        var el2 = dom.createTextNode("\n");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var element0 = dom.childAt(fragment, [6, 1]);
        var morphs = new Array(4);
        morphs[0] = dom.createMorphAt(fragment, 0, 0, contextualElement);
        morphs[1] = dom.createMorphAt(dom.childAt(fragment, [2]), 1, 1);
        morphs[2] = dom.createMorphAt(dom.childAt(fragment, [4]), 1, 1);
        morphs[3] = dom.createElementMorph(element0);
        dom.insertBoundary(fragment, 0);
        return morphs;
      },
      statements: [["inline", "link-to", ["Back home", "contacts"], [], ["loc", [null, [1, 0], [1, 34]]]], ["content", "model.name", ["loc", [null, [3, 18], [3, 32]]]], ["block", "link-to", ["edit", ["get", "model", ["loc", [null, [6, 20], [6, 25]]]]], [], 0, null, ["loc", [null, [6, 2], [8, 14]]]], ["element", "action", ["deleteContact", ["get", "model", ["loc", [null, [12, 35], [12, 40]]]]], [], ["loc", [null, [12, 10], [12, 42]]]]],
      locals: [],
      templates: [child0]
    };
  })());
});
define("dummy/templates/contacts", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    var child0 = (function () {
      return {
        meta: {
          "fragmentReason": {
            "name": "triple-curlies"
          },
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 1,
              "column": 0
            },
            "end": {
              "line": 3,
              "column": 0
            }
          },
          "moduleName": "dummy/templates/contacts.hbs"
        },
        isEmpty: false,
        arity: 0,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createTextNode("  ");
          dom.appendChild(el0, el1);
          var el1 = dom.createElement("p");
          dom.setAttribute(el1, "class", "error");
          var el2 = dom.createTextNode("There was an error: ");
          dom.appendChild(el1, el2);
          var el2 = dom.createElement("span");
          var el3 = dom.createComment("");
          dom.appendChild(el2, el3);
          dom.appendChild(el1, el2);
          dom.appendChild(el0, el1);
          var el1 = dom.createTextNode("\n");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
          var morphs = new Array(1);
          morphs[0] = dom.createMorphAt(dom.childAt(fragment, [1, 1]), 0, 0);
          return morphs;
        },
        statements: [["content", "error", ["loc", [null, [2, 45], [2, 54]]]]],
        locals: [],
        templates: []
      };
    })();
    var child1 = (function () {
      var child0 = (function () {
        return {
          meta: {
            "fragmentReason": false,
            "revision": "Ember@2.3.0",
            "loc": {
              "source": null,
              "start": {
                "line": 4,
                "column": 2
              },
              "end": {
                "line": 6,
                "column": 2
              }
            },
            "moduleName": "dummy/templates/contacts.hbs"
          },
          isEmpty: false,
          arity: 1,
          cachedFragment: null,
          hasRendered: false,
          buildFragment: function buildFragment(dom) {
            var el0 = dom.createDocumentFragment();
            var el1 = dom.createTextNode("    ");
            dom.appendChild(el0, el1);
            var el1 = dom.createElement("p");
            var el2 = dom.createComment("");
            dom.appendChild(el1, el2);
            dom.appendChild(el0, el1);
            var el1 = dom.createTextNode("\n");
            dom.appendChild(el0, el1);
            return el0;
          },
          buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
            var morphs = new Array(1);
            morphs[0] = dom.createMorphAt(dom.childAt(fragment, [1]), 0, 0);
            return morphs;
          },
          statements: [["inline", "link-to", [["get", "contact.name", ["loc", [null, [5, 17], [5, 29]]]], "contact", ["get", "contact", ["loc", [null, [5, 40], [5, 47]]]]], [], ["loc", [null, [5, 7], [5, 49]]]]],
          locals: ["contact"],
          templates: []
        };
      })();
      return {
        meta: {
          "fragmentReason": false,
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 3,
              "column": 0
            },
            "end": {
              "line": 7,
              "column": 0
            }
          },
          "moduleName": "dummy/templates/contacts.hbs"
        },
        isEmpty: false,
        arity: 0,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createComment("");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
          var morphs = new Array(1);
          morphs[0] = dom.createMorphAt(fragment, 0, 0, contextualElement);
          dom.insertBoundary(fragment, 0);
          dom.insertBoundary(fragment, null);
          return morphs;
        },
        statements: [["block", "each", [["get", "model", ["loc", [null, [4, 10], [4, 15]]]]], [], 0, null, ["loc", [null, [4, 2], [6, 11]]]]],
        locals: [],
        templates: [child0]
      };
    })();
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["wrong-type", "multiple-nodes"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 18,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/contacts.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createElement("hr");
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createElement("div");
        var el2 = dom.createTextNode("\n  ");
        dom.appendChild(el1, el2);
        var el2 = dom.createElement("div");
        var el3 = dom.createComment("");
        dom.appendChild(el2, el3);
        dom.appendChild(el1, el2);
        var el2 = dom.createTextNode("\n\n  ");
        dom.appendChild(el1, el2);
        var el2 = dom.createElement("div");
        var el3 = dom.createTextNode("\n    ");
        dom.appendChild(el2, el3);
        var el3 = dom.createElement("button");
        var el4 = dom.createTextNode("Create a new contact");
        dom.appendChild(el3, el4);
        dom.appendChild(el2, el3);
        var el3 = dom.createTextNode("\n  ");
        dom.appendChild(el2, el3);
        dom.appendChild(el1, el2);
        var el2 = dom.createTextNode("\n");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var element0 = dom.childAt(fragment, [4]);
        var element1 = dom.childAt(element0, [3, 1]);
        var morphs = new Array(3);
        morphs[0] = dom.createMorphAt(fragment, 0, 0, contextualElement);
        morphs[1] = dom.createMorphAt(dom.childAt(element0, [1]), 0, 0);
        morphs[2] = dom.createElementMorph(element1);
        dom.insertBoundary(fragment, 0);
        return morphs;
      },
      statements: [["block", "if", [["get", "error", ["loc", [null, [1, 6], [1, 11]]]]], [], 0, 1, ["loc", [null, [1, 0], [7, 7]]]], ["inline", "input", [], ["value", ["subexpr", "@mut", [["get", "newName", ["loc", [null, [12, 21], [12, 28]]]]], [], []], "placeholder", "Name"], ["loc", [null, [12, 7], [12, 49]]]], ["element", "action", ["createContact"], [], ["loc", [null, [15, 12], [15, 38]]]]],
      locals: [],
      templates: [child0, child1]
    };
  })());
});
define("dummy/templates/edit", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["multiple-nodes", "wrong-type"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 7,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/edit.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createElement("div");
        var el2 = dom.createTextNode("\n  ");
        dom.appendChild(el1, el2);
        var el2 = dom.createComment("");
        dom.appendChild(el1, el2);
        var el2 = dom.createTextNode("\n");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        var el1 = dom.createElement("button");
        var el2 = dom.createTextNode("Save");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var element0 = dom.childAt(fragment, [4]);
        var morphs = new Array(3);
        morphs[0] = dom.createMorphAt(dom.childAt(fragment, [0]), 1, 1);
        morphs[1] = dom.createMorphAt(fragment, 2, 2, contextualElement);
        morphs[2] = dom.createElementMorph(element0);
        return morphs;
      },
      statements: [["inline", "input", [], ["value", ["subexpr", "@mut", [["get", "model.name", ["loc", [null, [2, 16], [2, 26]]]]], [], []]], ["loc", [null, [2, 2], [2, 28]]]], ["inline", "link-to", ["Cancel", "contact", ["get", "model", ["loc", [null, [5, 29], [5, 34]]]]], [], ["loc", [null, [5, 0], [5, 36]]]], ["element", "action", ["saveContact", ["get", "model", ["loc", [null, [6, 31], [6, 36]]]]], [], ["loc", [null, [6, 8], [6, 38]]]]],
      locals: [],
      templates: []
    };
  })());
});
define("dummy/templates/friend", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    return {
      meta: {
        "fragmentReason": {
          "name": "triple-curlies"
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 2,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/friend.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createElement("h2");
        dom.setAttribute(el1, "class", "friend-name");
        var el2 = dom.createComment("");
        dom.appendChild(el1, el2);
        dom.appendChild(el0, el1);
        var el1 = dom.createTextNode("\n");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var morphs = new Array(1);
        morphs[0] = dom.createMorphAt(dom.childAt(fragment, [0]), 0, 0);
        return morphs;
      },
      statements: [["content", "model.name", ["loc", [null, [1, 24], [1, 38]]]]],
      locals: [],
      templates: []
    };
  })());
});
define("dummy/templates/friends", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    var child0 = (function () {
      return {
        meta: {
          "fragmentReason": false,
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 1,
              "column": 0
            },
            "end": {
              "line": 3,
              "column": 0
            }
          },
          "moduleName": "dummy/templates/friends.hbs"
        },
        isEmpty: false,
        arity: 1,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createTextNode("  ");
          dom.appendChild(el0, el1);
          var el1 = dom.createElement("p");
          var el2 = dom.createComment("");
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode(" is your ");
          dom.appendChild(el1, el2);
          var el2 = dom.createComment("");
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode("-year-old friend.");
          dom.appendChild(el1, el2);
          dom.appendChild(el0, el1);
          var el1 = dom.createTextNode("\n");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
          var element0 = dom.childAt(fragment, [1]);
          var morphs = new Array(2);
          morphs[0] = dom.createMorphAt(element0, 0, 0);
          morphs[1] = dom.createMorphAt(element0, 2, 2);
          return morphs;
        },
        statements: [["content", "friend.name", ["loc", [null, [2, 5], [2, 20]]]], ["content", "friend.age", ["loc", [null, [2, 29], [2, 43]]]]],
        locals: ["friend"],
        templates: []
      };
    })();
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["wrong-type"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 4,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/friends.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var morphs = new Array(1);
        morphs[0] = dom.createMorphAt(fragment, 0, 0, contextualElement);
        dom.insertBoundary(fragment, 0);
        dom.insertBoundary(fragment, null);
        return morphs;
      },
      statements: [["block", "each", [["get", "model", ["loc", [null, [1, 8], [1, 13]]]]], [], 0, null, ["loc", [null, [1, 0], [3, 9]]]]],
      locals: [],
      templates: [child0]
    };
  })());
});
define("dummy/templates/pets", ["exports"], function (exports) {
  exports["default"] = Ember.HTMLBars.template((function () {
    var child0 = (function () {
      return {
        meta: {
          "fragmentReason": {
            "name": "triple-curlies"
          },
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 1,
              "column": 0
            },
            "end": {
              "line": 3,
              "column": 0
            }
          },
          "moduleName": "dummy/templates/pets.hbs"
        },
        isEmpty: false,
        arity: 0,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createTextNode("  ");
          dom.appendChild(el0, el1);
          var el1 = dom.createElement("p");
          dom.setAttribute(el1, "class", "error");
          var el2 = dom.createTextNode("There was an error: ");
          dom.appendChild(el1, el2);
          var el2 = dom.createElement("span");
          var el3 = dom.createComment("");
          dom.appendChild(el2, el3);
          dom.appendChild(el1, el2);
          dom.appendChild(el0, el1);
          var el1 = dom.createTextNode("\n");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
          var morphs = new Array(1);
          morphs[0] = dom.createMorphAt(dom.childAt(fragment, [1, 1]), 0, 0);
          return morphs;
        },
        statements: [["content", "error", ["loc", [null, [2, 45], [2, 54]]]]],
        locals: [],
        templates: []
      };
    })();
    var child1 = (function () {
      var child0 = (function () {
        return {
          meta: {
            "fragmentReason": false,
            "revision": "Ember@2.3.0",
            "loc": {
              "source": null,
              "start": {
                "line": 5,
                "column": 4
              },
              "end": {
                "line": 7,
                "column": 4
              }
            },
            "moduleName": "dummy/templates/pets.hbs"
          },
          isEmpty: false,
          arity: 1,
          cachedFragment: null,
          hasRendered: false,
          buildFragment: function buildFragment(dom) {
            var el0 = dom.createDocumentFragment();
            var el1 = dom.createTextNode("      ");
            dom.appendChild(el0, el1);
            var el1 = dom.createElement("li");
            var el2 = dom.createElement("span");
            dom.setAttribute(el2, "class", "name");
            var el3 = dom.createComment("");
            dom.appendChild(el2, el3);
            dom.appendChild(el1, el2);
            var el2 = dom.createTextNode(" / alive: ");
            dom.appendChild(el1, el2);
            var el2 = dom.createElement("span");
            dom.setAttribute(el2, "class", "alive");
            var el3 = dom.createComment("");
            dom.appendChild(el2, el3);
            dom.appendChild(el1, el2);
            dom.appendChild(el0, el1);
            var el1 = dom.createTextNode("\n");
            dom.appendChild(el0, el1);
            return el0;
          },
          buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
            var element0 = dom.childAt(fragment, [1]);
            var morphs = new Array(2);
            morphs[0] = dom.createMorphAt(dom.childAt(element0, [0]), 0, 0);
            morphs[1] = dom.createMorphAt(dom.childAt(element0, [2]), 0, 0);
            return morphs;
          },
          statements: [["content", "pet.name", ["loc", [null, [6, 29], [6, 41]]]], ["content", "pet.alive", ["loc", [null, [6, 78], [6, 91]]]]],
          locals: ["pet"],
          templates: []
        };
      })();
      return {
        meta: {
          "fragmentReason": false,
          "revision": "Ember@2.3.0",
          "loc": {
            "source": null,
            "start": {
              "line": 3,
              "column": 0
            },
            "end": {
              "line": 11,
              "column": 0
            }
          },
          "moduleName": "dummy/templates/pets.hbs"
        },
        isEmpty: false,
        arity: 0,
        cachedFragment: null,
        hasRendered: false,
        buildFragment: function buildFragment(dom) {
          var el0 = dom.createDocumentFragment();
          var el1 = dom.createTextNode("  ");
          dom.appendChild(el0, el1);
          var el1 = dom.createElement("ul");
          var el2 = dom.createTextNode("\n");
          dom.appendChild(el1, el2);
          var el2 = dom.createComment("");
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode("    ");
          dom.appendChild(el1, el2);
          var el2 = dom.createComment("");
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode("\n    ");
          dom.appendChild(el1, el2);
          var el2 = dom.createElement("button");
          var el3 = dom.createTextNode("create");
          dom.appendChild(el2, el3);
          dom.appendChild(el1, el2);
          var el2 = dom.createTextNode("\n  ");
          dom.appendChild(el1, el2);
          dom.appendChild(el0, el1);
          var el1 = dom.createTextNode("\n");
          dom.appendChild(el0, el1);
          return el0;
        },
        buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
          var element1 = dom.childAt(fragment, [1]);
          var element2 = dom.childAt(element1, [5]);
          var morphs = new Array(3);
          morphs[0] = dom.createMorphAt(element1, 1, 1);
          morphs[1] = dom.createMorphAt(element1, 3, 3);
          morphs[2] = dom.createElementMorph(element2);
          return morphs;
        },
        statements: [["block", "each", [["get", "model", ["loc", [null, [5, 12], [5, 17]]]]], [], 0, null, ["loc", [null, [5, 4], [7, 13]]]], ["inline", "input", [], ["value", ["subexpr", "@mut", [["get", "petName", ["loc", [null, [8, 18], [8, 25]]]]], [], []]], ["loc", [null, [8, 4], [8, 27]]]], ["element", "action", ["createPet"], [], ["loc", [null, [9, 12], [9, 34]]]]],
        locals: [],
        templates: [child0]
      };
    })();
    return {
      meta: {
        "fragmentReason": {
          "name": "missing-wrapper",
          "problems": ["wrong-type"]
        },
        "revision": "Ember@2.3.0",
        "loc": {
          "source": null,
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 12,
            "column": 0
          }
        },
        "moduleName": "dummy/templates/pets.hbs"
      },
      isEmpty: false,
      arity: 0,
      cachedFragment: null,
      hasRendered: false,
      buildFragment: function buildFragment(dom) {
        var el0 = dom.createDocumentFragment();
        var el1 = dom.createComment("");
        dom.appendChild(el0, el1);
        return el0;
      },
      buildRenderNodes: function buildRenderNodes(dom, fragment, contextualElement) {
        var morphs = new Array(1);
        morphs[0] = dom.createMorphAt(fragment, 0, 0, contextualElement);
        dom.insertBoundary(fragment, 0);
        dom.insertBoundary(fragment, null);
        return morphs;
      },
      statements: [["block", "if", [["get", "error", ["loc", [null, [1, 6], [1, 11]]]]], [], 0, 1, ["loc", [null, [1, 0], [11, 7]]]]],
      locals: [],
      templates: [child0, child1]
    };
  })());
});
define('dummy/tests/mirage/mirage/config.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/config.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/config.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/config.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/config.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/config.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/factories/address.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/factories/address.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/address.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/factories/address.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/factories/address.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/address.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/factories/blog-post.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/factories/blog-post.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/blog-post.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/factories/blog-post.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/factories/blog-post.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/blog-post.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/factories/contact.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/factories/contact.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/contact.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/factories/contact.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/factories/contact.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/contact.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/factories/friend.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/factories/friend.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/friend.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/factories/friend.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/factories/friend.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/friend.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/factories/pet.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/factories/pet.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/pet.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/factories/pet.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/factories/pet.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/pet.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/factories/word-smith.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/factories/word-smith.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/word-smith.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/factories/word-smith.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/factories/word-smith.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/factories/word-smith.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/addresses.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/fixtures/addresses.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/addresses.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/addresses.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/fixtures/addresses.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/addresses.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/blog-posts.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/fixtures/blog-posts.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/blog-posts.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/blog-posts.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/fixtures/blog-posts.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/blog-posts.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/contacts.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/fixtures/contacts.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/contacts.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/contacts.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/fixtures/contacts.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/contacts.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/friends.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/fixtures/friends.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/friends.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/friends.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/fixtures/friends.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/friends.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/word-smiths.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/fixtures/word-smiths.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/word-smiths.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/fixtures/word-smiths.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/fixtures/word-smiths.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/fixtures/word-smiths.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/models/blog-post.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/models/blog-post.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/blog-post.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/models/blog-post.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/models/blog-post.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/blog-post.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/models/contact.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/models/contact.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/contact.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/models/contact.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/models/contact.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/contact.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/models/friend.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/models/friend.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/friend.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/models/friend.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/models/friend.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/friend.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/models/word-smith.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/models/word-smith.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/word-smith.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/models/word-smith.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/models/word-smith.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/models/word-smith.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/scenarios/default.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/scenarios/default.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/scenarios/default.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/scenarios/default.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/scenarios/default.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/scenarios/default.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/serializers/application.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/serializers/application.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/serializers/application.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/serializers/application.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/serializers/application.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/serializers/application.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/serializers/blog-post.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/serializers/blog-post.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/serializers/blog-post.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/serializers/blog-post.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/serializers/blog-post.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/serializers/blog-post.js should pass jshint.');
  });
});
define('dummy/tests/mirage/mirage/serializers/word-smith.jscs-test', ['exports'], function (exports) {
  QUnit.module('JSCS - mirage/serializers/word-smith.js');
  QUnit.test('should pass jscs', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/serializers/word-smith.js should pass jscs.');
  });
});
define('dummy/tests/mirage/mirage/serializers/word-smith.jshint', ['exports'], function (exports) {
  QUnit.module('JSHint - mirage/serializers/word-smith.js');
  QUnit.test('should pass jshint', function (assert) {
    assert.expect(1);
    assert.ok(true, 'mirage/serializers/word-smith.js should pass jshint.');
  });
});
/* jshint ignore:start */



/* jshint ignore:end */

/* jshint ignore:start */

define('dummy/config/environment', ['ember'], function(Ember) {
  var prefix = 'dummy';
/* jshint ignore:start */

try {
  var metaName = prefix + '/config/environment';
  var rawConfig = document.querySelector('meta[name="' + metaName + '"]').getAttribute('content');
  var config = JSON.parse(unescape(rawConfig));

  var exports = { 'default': config };

  Object.defineProperty(exports, '__esModule', { value: true });

  return exports;
}
catch(err) {
  throw new Error('Could not read config from meta tag with name "' + metaName + '".');
}

/* jshint ignore:end */

});

/* jshint ignore:end */

/* jshint ignore:start */

if (!runningTests) {
  require("dummy/app")["default"].create({"name":"ember-cli-mirage","version":"0.2.4+365a6332"});
}

/* jshint ignore:end */
